### DISCUSSION TITLE
export PUID="user-...":  Is the value of PGID the account name? Need to set a password?  I have multiple Chatgpt plus accounts, Can multiple accounts be set for PUID?

### DISCUSSION THREAD

**Discussion by luluhuhu** [acheong08/ChatGPT-Proxy-V4#82] [Q&A]
export PUID="user-...":

Is the value of PGID the account name?
Need to set a password?

I have multiple Chatgpt plus accounts,
Can multiple accounts be set for PUID?

**Comment by acheong08** [comment]
PUID is a cookie found in browser.
